pkg/virt-controller/watch/vmi/volume-hotplug.go

```diff
@@ -73,6 +73,20 @@ func getActiveAndOldAttachmentPods(readyHotplugVolumes []*v1.Volume, hotplugAtta
 // pod that is closest to the desired state.
 func (c *Controller) cleanupAttachmentPods(currentPod *k8sv1.Pod, oldPods []*k8sv1.Pod, vmi *v1.VirtualMachineInstance, numReadyVolumes int) common.SyncError {
 	foundRunning := false
+
+	var statusMap = make(map[string]v1.VolumeStatus)
```

Comment:
Do I understand correctly that this map holds all "transient" volumes that could also potentially be deleted, because they are no longer in the spec?

Reply:
correct, these are volumes that are pending delete